platforms/defaults.go

```diff
@@ -16,6 +16,7 @@ func DefaultSpec() specs.Platform {
 	return specs.Platform{
 		OS:           runtime.GOOS,
 		Architecture: runtime.GOARCH,
-		// TODO(stevvooe): Need to resolve GOARM for arm hosts.
+		// The variant field whould be empty if the arch is not arm family.
```

Comment:
There is a typo in the comment, but let's change the comment to "*the variant field will be empty if arch != ARM*"

Reply:
Ok, I will update it in next version